doc/visual-programming/source/widgets/unsupervised/PCA.rst

```diff
@@ -38,9 +38,12 @@ instances or weights of principal components.
 1. Select how many principal components you wish in your output. It is
    best to choose as few as possible with variance covered as high as possible. You can also set how much variance you
    wish to cover with your principal components.
-2. When *Auto send on change* is ticked, the widget will automatically
-   communicate all the changes. Alternatively click *Send data*.
-3. Principal components graph, where the red (lower) line is the
+2. You can normalize data to minimize data redundancy. 
```

Comment:
> minimize data redundancy

Hmmm... Is this correct? I'm no expert here, but I think it should just be: to adjust the values to common scale. Should be unchecked for, say, gene expression data.